java/integTest/src/test/java/glide/PubSubTests.java

```diff
@@ -128,17 +131,23 @@ private BaseClient createClient(boolean standalone) {
 
     private static final int MESSAGE_DELIVERY_DELAY = 500; // ms
 
-    @BeforeEach
+    @AfterEach
```

Comment:
lol

Reply:
I don't know why, but is never called with `@BeforeEach`.